evalml/tests/component_tests/test_imputer.py

```diff
@@ -521,53 +521,87 @@ def test_imputer_does_not_erase_ww_info():
     mock_transform.call_args[0][0].ww.schema == df_holdout.ww[["b"]].ww.schema
 
 
+def get_int_df():
```

Comment:
omega nit-pick / general curiosity but should these be fixtures? (not really sure if there's a clear yay or nay here, just wanted to raise awareness since that's how we've organized our other dummy datasets but it doesn't matter to me either way)

Reply:
Cool, so I just moved up `test_imputer`'s imputer_test_data to `conftest.py` and I use that instead.  Sniped these garbage data frames as I figured I should drink my own koolaid and practice what I preach 😂